packages/fe-container/src/containers/launcher/Launcher.less

```diff
@@ -13,3 +13,9 @@
   padding: 0 24px !important;
   border-bottom: 1px solid #d9d9d9;
 }
+
+.launcher_footer {
+  z-index: 1000;
```

Comment:
IMHO I think that this should not be above the side menu, but make a check with Veronica

Reply:
Veronica said that it should be above being a banner.